python/ngraph/ops.py

```diff
@@ -274,49 +282,77 @@ def dot(left_node, right_node, name=None):
 
 # convpool ops
 @nameable_op
-def convolution(x,                      # type: Node
```

Comment:
Let's verify that names here are identical (or similar) to ngraph++ API names.

Reply:
Updated.